Makefile

```diff
@@ -108,13 +111,13 @@ ci-tests:\
 
 ## Actual Rules (Travis)
 
-.PHONY: ci-formatting
-ci-formatting:
-	@printf "$$(tput bold)******************$$(tput sgr0)\n"
-	@printf "$$(tput bold)* CI: Formatting *$$(tput sgr0)\n"
-	@printf "$$(tput bold)******************$$(tput sgr0)\n"
-	@CI=true ./tools/run_cargo_fmt.sh diff
-	@./tools/check_wildcard_imports.sh
+.PHONY: ci-lints
```

Comment:
I don't see this rule used anywhere. Is it needed?

Reply:
Oh, good catch, I meant for it to be used by Travis.